public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx

```diff
@@ -80,7 +79,6 @@ export const SingleTopBar = memo(function SingleTopBar({
             <ToolbarButton iconOnly icon="question-circle" aria-label="Help" />
           </Dropdown>
         )}
-        {config.newsFeedEnabled && <NewsContainer />}
```

Comment:
this is the only line you need to remove, and then we need to add it back somewhere else

Reply:
fixed. force pushed it. 

I guess we can leave the setting in case its useful for the new placement but Im worried that it will just stick around and be annoying.

Reply:
@bergquist need to figure out where to add it back to

Reply:
Should we move support under profile as well? cc @mjseaman @thanos-karachalios

Reply:
@bergquist I wouldn't move the support button. It does have a low click rate similar to the newsfeed, but it contains some necessary info, like the version or release channel, it has utility on every page, and it isn't a user setting.

Reply:
> and it isn't a user setting.

neither is news :) But I agree that support is important enough to take space in the topnav.